tests/files/test-image.php

```diff
@@ -76,7 +76,7 @@ protected static function getMethod( $name ) {
 	 *
 	 * @return ReflectionProperty
 	 */
-	protected function getProperty( $name ) {
+	protected function get_property( $name ) {
```

Comment:
The `AssertAttributeHelper` trait from `yoast/phpunit-polyfills` adds two static methods: `getProperty()` and `getPropertyValue()`. The first one conflicts with the definition of our `getProperty()`. The implementation of both methods is incompatible; this makes it very difficult to fix `test-image.php` to use the pollyfilled methods.

I, therefore, decided to rename the methods using the snake case (this seems to be in line with the coding standards).

Reply:
It makes sense. Could a brief explanation of this be added in the method's comment? Something like `Renaming to get_property to avoid conflict with polyfills`?

Reply:
Done in 1c7eadd6f